data/regions.csv

```diff
@@ -10,3 +10,4 @@ E12000008,South East
 E12000009,South West
 E13000001,Inner London
 E13000002,Outer London
+z,Outside of England and unknown
```

Comment:
Are we really expecting the label to be 'Outside of England and unknown'? Should they not be split and just both share the z code?

Reply:
Yes, that is the specific aggregation they have always used with the name of 'Other'. Even if they did split it up (which we discussed) they should still supply the combination of them all as users expect to see it.